example/package.json

```diff
@@ -5,6 +5,7 @@
   "private": true,
   "license": "MIT",
   "dependencies": {
+    "font-awesome": "^4.7.0",
```

Comment:
nit: Looks like you forgot to update the Yarn lock file.

That being said, I'd like to avoid pulling in a dependency on font-awesome's stylesheets if avoidable. What is this giving us besides an arrow icon? (Can we just make our own?)

Reply:
As a side note, I kind of prefer the hamburger (3 horizontal line) icon style to the spinning arrow anyway 😄 We could probably just snag this from Google's Material SVGs.